models/reviews.models.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../db/connection');
+
+exports.selectReview = (id_param) => {
+    const { review_id } = id_param;
+    if (isNaN(parseInt(review_id))) {
```

Comment:
You shouldn't need to throw this error manually - PSQL will throw an error for you as the review_id is the wrong data type. You'd then be able to see the error in your error handler if you console.log(err) on line 13 of your app.js, which is where you'd send the status and message for this kind of error.